From 2b0434ae7a2cbcbe315bdc1522bf74dc7c534071 Mon Sep 17 00:00:00 2001
From: Sebastian Reichel <sre@kernel.org>
Date: Wed, 2 Mar 2016 16:56:52 +0100
Subject: [PATCH 33/53] drm: omapdrm: crtc: add enabled bit to state

Signed-off-by: Sebastian Reichel <sre@kernel.org>
---
 drivers/gpu/drm/omapdrm/omap_crtc.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/drivers/gpu/drm/omapdrm/omap_crtc.c b/drivers/gpu/drm/omapdrm/omap_crtc.c
index 892aaf120c5a..70081a5e8f7f 100644
--- a/drivers/gpu/drm/omapdrm/omap_crtc.c
+++ b/drivers/gpu/drm/omapdrm/omap_crtc.c
@@ -209,6 +209,8 @@ static int omap_crtc_dss_enable(enum omap_channel channel)
			&omap_crtc->vm);
	omap_crtc_set_enabled(&omap_crtc->base, true);

+	set_bit(crtc_enabled, &omap_crtc->state);
+
	return 0;
 }

@@ -216,6 +218,8 @@ static void omap_crtc_dss_disable(enum omap_channel channel)
 {
	struct omap_crtc *omap_crtc = omap_crtcs[channel];

+	clear_bit(crtc_enabled, &omap_crtc->state);
+
	omap_crtc_set_enabled(&omap_crtc->base, false);
 }

--
2.11.0
